app/models/report_generators/system_performance/fy2019/base.rb

```diff
@@ -134,5 +134,16 @@ def finish_report
         completed_at: Time.now
       )
     end
+
+    def personal_ids(destination_ids)
+      sc_t = GrdaWarehouse::Hud::Client.arel_table.dup
+      sc_t.table_alias = 'source_clients_Client' # Relies on how AR names the inner join
+
+      GrdaWarehouse::Hud::Client.joins(:source_clients).
+        where(id: destination_ids).
+        select(:id, sc_t[:PersonalID]).
+        pluck(:id, sc_t[:PersonalID]).
+        group_by(&:first).transform_values{ |v| v.map(&:last) }
```

Comment:
I think this would be much cleaner written as `GrdaWarehouse::WarehouseClient.joins(:source).where(destination_id: destination_ids).pluck(: destination_id, c_t[:PersonalID])....